.github/workflows/ms.keyvault.vaults.yml

```diff
@@ -31,6 +31,8 @@ env:
   AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
   ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
   ARM_MGMTGROUP_ID: ${{ secrets.ARM_MGMTGROUP_ID }}
+  ARM_PRINCIPAL_ID: ${{ secrets.ARM_PRINCIPAL_ID }}
```

Comment:
I guess you added just a few examples but we'd need to add this added references to all workflows, right?

Reply:
Correct. I wanted to ask first if that is our only way in getting the secrets in. Maybe it is part of the design that actions should not be able to fetch secrets unless you want to feed it to them (i.e. Input or Env variable). Just needed to make sure we're happy with the approach before I flip it to all workflows

Reply:
It's the only way I am aware of, yes. Unfortunate, but necessary if not using a plain json.